lib/Syntax/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 set(line_directive "#line" "%(line)d" "\"%(file)s\"")
+
 set(SWIFT_GYB_FLAGS
-  --line-directive "'${line_directive}'")
+  --line-directive "\"${line_directive}\"")
```

Comment:
I don't think this is correct; note that the actual body of `${line_directive}` contains quotes.

Reply:
@AndrewSB you might need to double-escape the quotes.

`”\\\”%(file)s\\\””`

Reply:
parsing on parsing 😂

Thanks you two, I'll make the change now, and I can test it tomorrow during the day, when I'm back at my Windows machine